app/import.go

```diff
@@ -817,6 +817,12 @@ func (a *App) ImportUserTeams(user *model.User, data *[]UserTeamImportData) *mod
 			}
 		}
 
+		if defaultChannel, err := a.GetChannelByName(model.DEFAULT_CHANNEL, team.Id); err != nil {
```

Comment:
I can’t remember but I think there is a function to add the user to all defaults

Reply:
Perhaps take a look at the api for creating a user

Reply:
Yes, there is `JoinDefaultChannels` but I'm not sure it is something that must be done. I mean, the `JoinDefaultChannels` join the user to `town-square` and `off-topic` both are default, but only `town-square` is mandatory. If you import data, you want to respect the imported data by the user as much as possible, as far as they don't break the system rules, in this case, the `town-square` mandatory membership.

Reply:
Sounds good